sql/updates/world/2016_03_01_NPC_THALORIEN_DAWNSEEKER_world.sql

```diff
@@ -0,0 +1,79 @@
+-- Thalorien Dawnseeker's Remains
+SET @ENTRY := 37552;
+
+DELETE FROM `creature_template_addon` WHERE (`entry`=@ENTRY);
+INSERT INTO `creature_template_addon` (`entry`,`path_id`,`mount`,`bytes1`,`bytes2`,`emote`,`auras`) VALUES 
+(@ENTRY, 0, 0, 0, 0, 0, '25824 31261');
```

Comment:
congrats you have posted on the PR of one user who deleted the account and a work never merged.